name: Supervised Classification, features: MiceProtein
{'X_train': {'cols': 0.016371831186646004, 'rows': 0.48010973936899864}, 'X_valid': {'cols': 0.01327088212334114, 'rows': 0.43169398907103823}, 'X_test': {'cols': 0.018518518518518517, 'rows': 0.5222222222222223}}
dataset sizes
(768, 77) (183, 77) (270, 77)
dropped dataset sizes
(402, 77) (104, 77) (129, 77)
key: 1896, k: 1/4, dataset: 40966, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 14, 'max_bin': 139, 'max_depth': 9, 'min_data_in_leaf': 14, 'learning_rate': 0.05444883811218653, 'num_iterations': 633, 'objective': 'softmax', 'verbose': -1, 'num_class': 8}
Training until validation scores don't improve for 100 rounds
Early stopping, best iteration is:
[451]	valid_0's multi_logloss: 0.0528903
strategy:None, acc gbm: 0.9740740740740741
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
strategy:None, nll xbg:0.06410405039787292
name: Supervised Classification, features: MiceProtein
{'X_train': {'cols': 0.017450155478324493, 'rows': 0.49014084507042255}, 'X_valid': {'cols': 0.016999854078505763, 'rows': 0.46629213483146065}, 'X_test': {'cols': 0.014718614718614718, 'rows': 0.48148148148148145}}
dataset sizes
(744, 77) (178, 77) (270, 77)
dropped dataset sizes
(374, 77) (95, 77) (140, 77)
key: 1493, k: 2/4, dataset: 40966, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 14, 'max_bin': 139, 'max_depth': 9, 'min_data_in_leaf': 14, 'learning_rate': 0.05444883811218653, 'num_iterations': 633, 'objective': 'softmax', 'verbose': -1, 'num_class': 8}
Training until validation scores don't improve for 100 rounds
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
Early stopping, best iteration is:
[451]	valid_0's multi_logloss: 0.0476995
strategy:None, acc gbm: 0.9777777777777777
strategy:None, nll xbg:0.053086742758750916
name: Supervised Classification, features: MiceProtein
{'X_train': {'cols': 0.015873015873015872, 'rows': 0.46382428940568476}, 'X_valid': {'cols': 0.015798634355335385, 'rows': 0.5309278350515464}, 'X_test': {'cols': 0.017075517075517074, 'rows': 0.4703703703703704}}
dataset sizes
(848, 77) (194, 77) (270, 77)
dropped dataset sizes
(460, 77) (91, 77) (143, 77)
key: 3460, k: 3/4, dataset: 40966, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 14, 'max_bin': 139, 'max_depth': 9, 'min_data_in_leaf': 14, 'learning_rate': 0.05444883811218653, 'num_iterations': 633, 'objective': 'softmax', 'verbose': -1, 'num_class': 8}
Training until validation scores don't improve for 100 rounds
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
Early stopping, best iteration is:
[504]	valid_0's multi_logloss: 0.0337057
strategy:None, acc gbm: 0.9740740740740741
strategy:None, nll xbg:0.07337658107280731
name: Supervised Classification, features: MiceProtein
{'X_train': {'cols': 0.016216120835686056, 'rows': 0.4904891304347826}, 'X_valid': {'cols': 0.016163184641445512, 'rows': 0.4945652173913043}, 'X_test': {'cols': 0.01683501683501683, 'rows': 0.48148148148148145}}
dataset sizes
(768, 77) (184, 77) (270, 77)
dropped dataset sizes
(391, 77) (93, 77) (140, 77)
key: 1659, k: 4/4, dataset: 40966, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 14, 'max_bin': 139, 'max_depth': 9, 'min_data_in_leaf': 14, 'learning_rate': 0.05444883811218653, 'num_iterations': 633, 'objective': 'softmax', 'verbose': -1, 'num_class': 8}
Training until validation scores don't improve for 100 rounds
Early stopping, best iteration is:
[370]	valid_0's multi_logloss: 0.0489119
strategy:None, acc gbm: 0.9703703703703703
strategy:None, nll xbg:0.08605387806892395
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")